process/process_darwin.go

```diff
@@ -94,28 +94,37 @@ func (p *Process) NameWithContext(ctx context.Context) (string, error) {
 }
 
 func (p *Process) CmdlineWithContext(ctx context.Context) (string, error) {
-	r, err := callPsWithContext(ctx, "command", p.Pid, false)
+	r, err := callPsWithContext(ctx, "command", p.Pid, false, false)
 	if err != nil {
 		return "", err
 	}
 	return strings.Join(r[0], " "), err
 }
 
+// CmdNameWithContext returns the command name (including spaces) without any arguments
+func (p *Process) CmdNameWithContext(ctx context.Context) ([]string, error) {
```

Comment:
Could you change this function name to lower camel in order to not expose ?

Reply:
Done and thanks for the review. Sorry to cause trouble, I'm relatively new to golang.